vm/tests/executor.rs

```diff
@@ -815,34 +795,14 @@ fn timelock_block_age_fail() {
     )
 }
 
-#[test]
-fn timelock_block_age_success() {
```

Comment:
and it?

Reply:
It seems that it's a valid timelock argument.

Reply:
Fixed it! : )